File: Dev/Code/Community.Entity/Model/Area/AreaEntity.cs
using System;
using System.ComponentModel.DataAnnotations;
using YooPoon.Core.Data;

namespace Community.Entity.Model.Area
{
	public class AreaEntity : IBaseEntity
	{
		/// <summary>
		/// ID
		/// </summary>
		public virtual int Id { get; set; }
		/// <summary>
		/// 编号ID
		/// </summary>
		public virtual string CodeId { get; set; }
		/// <summary>
		/// 创建时间
		/// </summary>
		public virtual DateTime AddDate { get; set; }
		/// <summary>
		/// 父级ID
		/// </summary>
		public virtual AreaEntity Parent { get; set; }
		/// <summary>
		/// 区位名
		/// </summary>
		public virtual string Name { get; set; }
	}
}

File: Dev/Android/YoopoonMarket/src/com/yoopoon/market/domain/AreaEntity.java
package com.yoopoon.market.domain;

public class AreaEntity {

	public int Id;
	public String Codeid;
	public String Adddate;
	public String Parent;
	public String Name;
	public String ParentName;
	
	
	@Override
	public String toString() {
		return "AreaEntity [Id=" + Id + ", Codeid=" + Codeid + ", Adddate=" + Adddate + ", Parent=" + Parent
				+ ", Name=" + Name + ", ParentName=" + ParentName + "]";
	}

	

}
